scss/_variables.scss

```diff
@@ -1193,9 +1195,11 @@ $form-check-input-checked-color:          $component-active-color !default;
 $form-check-input-checked-bg-color:       $component-active-bg !default;
 $form-check-input-checked-border-color:   $form-check-input-checked-bg-color !default;
 $form-check-input-checked-bg-image:       var(--#{$prefix}check-icon) !default;
-$form-check-input-disabled-color:         var(--#{$prefix}form-check-input-disabled-color) !default; // Boosted mod
+$form-check-input-disabled-color:         var(--#{$prefix}form-check-disabled-color) !default; // Boosted mod
 $form-check-input-disabled-filter:        var(--#{$prefix}form-check-filter) !default; // Boosted mod
 $form-check-radio-checked-bg-image:       url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='2' fill='#{$form-check-input-checked-color}'/></svg>") !default;
+$form-check-filter:                       $invert-filter !default; // Boosted mod
```

Comment:
These 2 should be at the beginning of the `$form-check*` var declarations because they are less specific.